### ISSUE TITLE
[Feature Request]: Youden's J point

### ISSUE THREAD

**Issue by aboyle57** [jasp-stats/jasp-issues#2966]
### Description

Functionality to calculate Youden's J point in data to identify the optimal value of a biomarker to predict a binary outcome

### Use-case

Evaluating biomarker performance (as a threshold) in predicting mortality

### Is your feature request related to a JASP module?

Learn Bayes

### Describe the solution you would like

An option to automatically calculate Youden's J point

**Comment by tomtomme** [comment]
@aboyle57 
thx for the request. Is this something for the logistic regression module? Can you add  details / papers / R-Packages?

**Comment by tomtomme** [comment]
@aboyle57 
Thx. I see, this is then for the binary classification module, as you requested. The paper is certainly helpful.

**Comment by aboyle57** [comment]
@tomtomme Great. Do you think it's something that could be incorporated? It would be a neat addition to the ROC curve section.

I tried to run the R script in the R module within JASP but I couldn't get it to work, presumably because that functionality doesn't exist!

**Comment by tomtomme** [comment]
@aboyle57 
yeah sometimes not all needed R-packages are there. Or you need to select a different jasp module from within the R-terminal of JASP beforehand. 
either way, I can give no timeline, but it is now on the list
